client/src/utils_links.js

```diff
@@ -0,0 +1,144 @@
+import React from "react"
+import { gql } from "apollo-boost"
+import parse from "html-react-parser"
+import API from "api"
+import LinkAnet from "components/editor/LinkAnet"
+
+const UUID_REGEX =
+  "^[0-9a-zA-Z]{8}-[0-9a-zA-Z]{4}-[0-9a-zA-Z]{4}-[0-9a-zA-Z]{4}-[0-9a-zA-Z]{12}$"
+
+const GQL_GET_REPORT = gql`
+  query($uuid: String!) {
+    report(uuid: $uuid) {
+      uuid
+      intent
+    }
+  }
+`
+
+const GQL_GET_PERSON = gql`
+  query($uuid: String!) {
+    person(uuid: $uuid) {
+      uuid
+      name
+      role
+      avatar(size: 32)
+    }
+  }
+`
+
+const GQL_GET_ORGANIZATION = gql`
+  query($uuid: String!) {
+    organization(uuid: $uuid) {
+      uuid
+      shortName
+    }
+  }
+`
+
+const GQL_GET_POSITION = gql`
+  query($uuid: String!) {
+    position(uuid: $uuid) {
+      uuid
+      name
+    }
+  }
+`
+
+const GQL_GET_LOCATION = gql`
+  query($uuid: String!) {
+    location(uuid: $uuid) {
+      uuid
+      name
+    }
+  }
+`
+
+const GQL_GET_TASK = gql`
+  query($uuid: String!) {
+    task(uuid: $uuid) {
+      uuid
+      shortName
+      longName
+    }
+  }
+`
+
+export const ENTITY_TYPES = {
+  REPORT: "report",
+  PERSON: "person",
+  ORGANIZATION: "organization",
+  POSITION: "position",
+  LOCATION: "anetLocation",
+  TASK: "task"
```

Comment:
You should use the task label from the dictionary in the Settings.

Reply:
I think it would be a better alternative to use `getModelNameLinkTo` from the corresponding Model entity, as what I am trying to achieve is linking entity types to url links.

Reply:
Apparently I was responding to an old commit. This comment was about labels, which should indeed come from the dictionary.

Reply:
I actually ended up reusing the labels from SEARCH_OBJECT_LABELS (index.js) instead, as this dictionary was being use for 2 different purposes: labeling and object construction.